### PR TITLE
feat(sdk-middleware-http): allow to pass credentials mode to fetch object

### PR DESCRIPTION
**Pull request by emmenko**
#### Summary
When working with HTTP Cookies, the SDK HTTP middleware should be able to be configured to set the `credentials` mode (https://github.com/github/fetch#sending-cookies).

#### Description
Adds a new option `credentialsMode` to the `createHttpMiddleware`.

NOTE: I wasn't able to properly write a test for this, it's kind of tricky to reach into the internals of `fetch`.

### CODE REVIEW

**Comment by emmenko** [comment]
It's called credentials mode in the spec

https://fetch.spec.whatwg.org/#cors-protocol-and-credentials

**Comment by tdeekens** [comment]
Yup, saw that too after some Googling all good!

File: docs/sdk/api/sdkMiddlewareHttp.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Creates a [middleware](/sdk/Glossary.md#middleware) to handle HTTP requests for
 #### Named arguments (options)
 
 1. `host` *(String)*: the host of the HTTP API service
+2. `credentialsMode` *(String)*: one of the supported `credentials` modes (`omit`, `same-origin`, `include`), useful when working with HTTP Cookies. (Default: undefined)
```
Comment (wizzy25) [inline docs/sdk/api/sdkMiddlewareHttp.md]: I don't think the `(Default: undefined)` part is necessary. Makes it seem like `undefined` is a usable mode

Comment (emmenko) [reply docs/sdk/api/sdkMiddlewareHttp.md]: Yes good point, changed that to `optional` https://github.com/commercetools/nodejs/pull/485/commits/80552b4885b1c4cda35a148e34b06411311fb96a

**Comment by emmenko** [comment]
@wizzy25 would you like to release it please? 🙏

**Pull request merged**